dataplug-fitbit/app/org/hatdex/dataplugFitbit/Module.scala

```diff
@@ -62,6 +62,11 @@ class Module extends AbstractModule with ScalaModule with AkkaGuiceSupport {
     fitbitSleepInterface: FitbitSleepInterface,
     fitbitWeightInterface: FitbitWeightInterface,
     fitbitLifetimeStatsInterface: FitbitLifetimeStatsInterface,
+    FitbitSleepGoalsInterface: FitbitSleepGoalsInterface,
```

Comment:
Oh well. Just saw those. Why is it capital? Same below. How does that compile and work.... Sometimes Scala magic is beyond magical and that's not good

Reply:
It's just a variable name the developer comes up with. If you came up with a name that starts with a capital, it's a valid variable name. Why should it not compile? Now if you tried to start it with digit or another symbol...